src/tests/ftest/recovery/pass_1.py

```diff
@@ -0,0 +1,416 @@
+"""
+  (C) Copyright 2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+from ClusterShell.NodeSet import NodeSet
+from avocado.core.exceptions import TestFail
+
+from apricot import TestWithServers
+from general_utils import report_errors, pcmd, check_file_exists
+
+
+class Pass1Test(TestWithServers):
+    """Test Pass 1: Pool List Consolidation
+
+    :avocado: recursive
+    """
+
+    def test_dangling_pool(self):
+        """Test dangling pool.
+
+        1. Create a pool.
+        2. Remove the pool from the pool service (PS) on engine by calling:
+        dmg faults pool-svc <pool> CIC_POOL_NONEXIST_ON_ENGINE
+        3. Show dangling pool entry by calling:
+        dmg pool list --no-query
+        4. Enable and start the checker.
+        5. Query the checker and verify the message regarding dangling pool.
+        6. Disable the checker.
+        7. Verify that the dangling pool was removed. Call dmg pool list and it should
+        return an empty list.
+        8. Un-register the pool so that we don't try to destroy during tearDown.
+
+        Jira ID: DAOS-11711
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=recovery,pass_1
+        :avocado: tags=test_dangling_pool
```

Comment:
Recommended to also use class name for all tests in the class:
```suggestion
        :avocado: tags=Pass1Test,test_dangling_pool
```

Reply:
Done